docusaurus/docs/iOS/components/message-composer.md

```diff
@@ -69,41 +66,39 @@ class CustomMessageListVC: UIViewController {
+        keyboardHandler.start()
     }
 
     override open func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
 
-        // Remove the keyboard observer
-        keyboardObserver.unregister()
+        keyboardHandler.stop()
     }
 }
 ```
-As you can see if you want to use the `ComposerVC` in your own message list view you need to setup the dependencies of the composer, add it as a child view controller of your custom message list view controller and even manage the keyboard yourself or use our keyboard observer to manage it.
+As you can see if you want to use the `ComposerVC` in your custom message list view you need to setup the dependencies of the composer, add it as a child view controller of your custom message list view controller, and even manage the keyboard yourself or use our keyboard observer to manage it.
 
 ### Customization
 
-The `ComposerVC` and `ComposerView` are completely customizable. You can not only change the UI layout and styling, but you can extend the composer functionality as well. In case you want to change the styling, adding new views and new functionality you can take a look at the [Customize Message Composer](../guides/customize-message-composer) guide. If you want to add a new custom attachment and make the composer to support it, you should read the [Message Composer Custom Attachments](../guides/working-with-custom-attachments) guide. 
+The `ComposerVC` and `ComposerView` are completely customizable. You can not only change the UI layout and styling, but you can extend the composer functionality as well. In case you want to change the styling, adding new views, and new functionality you can take a look at the [Customize Message Composer](../guides/customize-message-composer) guide. If you want to add a new custom attachment and make the composer support it, you should read the [Message Composer Custom Attachments](../guides/working-with-custom-attachments) guide.
 
 ### Properties
 
-Complete list of all the components of `ComposerVC`.
+The complete list of all the `ComposerVC`'s components.
 
 <ComposerProperties/>
 
 ## Composer View
 
-The `ComposerView` class which holds all the composer subviews and implements the composer layout. The composer layout is built with multiple `ContainerStackView`'s, which are very similar how  `UIStackView`'s work, you can read more about them [here](../customization/custom-components#setuplayout). This makes it very customizable since to change the layout you only need to move/remove/add views from different containers.
+The `ComposerView` class holds all the composer subviews and implements the composer layout. The composer layout is built with multiple `ContainerStackView`'s, which are very similar how `UIStackView`'s work, you can read more about them [here](../customization/custom-components#setuplayout). This makes it very customizable since to change the layout you only need to move/remove/add views from different containers.
```

Comment:
```suggestion
The `ComposerView` class holds all the composer subviews and implements the composer layout. The composer layout is built with multiple `ContainerStackView`'s, which work similar to `UIStackView`'s. This makes it very customizable since to change the layout you only need to move/remove/add views to different containers. Read more about `ContainerStackView` [here](../customization/custom-components#setuplayout).
```